src/content/development/containers/file-sync/aspnetcore.mdx

```diff
@@ -77,6 +79,7 @@ The `hello-world` key matches the name of the hello world Deployment. The meanin
 - `command`: the start command of the development container.
 - `environment`: the environment variables added or overwritten in your development container.
 - `remote`: the local port to use for SSH communication with your development environment.
+- `sync`: which path is going to be synchorinized from local to container.
```

Comment:
@pchico83 Did you mean this section? I can update with the more used text you shared.

Reply:
oh yes! let's use the same text than the other samples